### ISSUE TITLE
Improve UI

### ISSUE THREAD

**Issue by narayan954** [narayan954/dummygram#4]
UI of the project can be improved to a great extent. 
We can start with parts.
This issue can become a thread for issues related to UI enhancement issues.
Contribution is welcome.

**Comment by ArshErgon** [comment]
assign it to me

**Comment by narayan954** [comment]
done @ArshErgon

**Comment by narayan954** [comment]
hey @ArshErgon I have decided that this issue is actually a parent issue for the many child issues, like improve UI for cards, improve UI for upload menu, etc , you can get yourself assigned in those child issues. And this issues will exist isolated.

**Comment by ArshErgon** [comment]
Hey @narayan954 no problem, i will improve everything which I can, the fonts or the UI, I haven't started the project yet, because I'm quite busy with some othe projects but will be starting the project tomorrow

**Comment by narayan954** [comment]
Yeah please propose the changes in a separate issue, so I can assign you. Also, no hurries

**Comment by Furkangunduz** [comment]
Do you have any design to do ? Or can we just do our best to solve something ?  I can work on sign up and sign in section if there is no one is working on.

**Comment by narayan954** [comment]
We can definitely improve the new post button, sign-up, sign-in interface! Feel free to create the issue! :)

**Comment by Kritika30032002** [comment]
Is this still available for contribution?

**Comment by narayan954** [comment]
> Is this still available for contribution?

yes it is :)
Ps - Create a separate issue specifying the elements you'll improve .

**Comment by MaYaNkKashyap681** [comment]
Can I work on this Issue?

**Comment by narayan954** [comment]
> Can I work on this Issue?

yes, please create an issue regarding the part you'd like to improve :)
Thanks

**Comment by MaYaNkKashyap681** [comment]
> > Can I work on this Issue?
> 
> yes, please create an issue regarding the part you'd like to improve :) Thanks

I want to improve the signing and signup form along with that card card components.

**Comment by narayan954** [comment]
> > > Can I work on this Issue?
> > 
> > 
> > yes, please create an issue regarding the part you'd like to improve :) Thanks
> 
> I want to improve the signing and signup form along with that card components.

I think there's already an issue regarding signing and signup form, you can create an issue for card components though, and please mention your program in that :)
Regards

**Comment by Aarav238** [comment]
i would like to work on it @narayan954

**Comment by narayan954** [comment]
> i would like to work on it @narayan954

Kindly create a separate issue indicating the part you'd like to work on and program you're associated with(iwoc or jwoc). 
Thanks

**Comment by sandeeprawat28** [comment]
Please assign this issue to me.

**Comment by narayan954** [comment]
follow the previous comment @sandeeprawat28

**Comment by Code1705** [comment]
colour and background needs to be  updated, please assign me this issue under JWOC , I will be happy enough to contribute.

**Comment by narayan954** [comment]
> colour and background needs to be updated, please assign me this issue under JWOC , I will be happy enough to contribute.

create the issue acknowledging this and claim it! @Code1705

**Comment by Code1705** [comment]
ok , definately

**Comment by 1010varun** [comment]
@narayan954 can I make a new issue for changing the UI page of signup and sign in page.

**Comment by narayan954** [comment]
> @narayan954 can I make a new issue for changing the UI page of signup and sign in page.

Of course you can! and it's recommended.

**Comment by deepbeatz** [comment]
I want to work on this issue under JWOC

**Comment by narayan954** [comment]
> I want to work on this issue under JWOC

follow the instructions

**Comment by narayan954** [comment]
You can be assigned to only one issue at a time, finish the assigned issue first. Thank you

**Comment by Hritik1503** [comment]
Please assign me this issue currently i am completed my all so i am free for this so please assign me this issue

**Comment by JatinBisht2308** [comment]
@narayan954 sir can I work on this issue under gssoc23.

**Comment by narayan954** [comment]
> @narayan954 sir can I work on this issue under gssoc23.

Yeah sure Jatin, create an issue mentioning the part you'll make contribution on.

**Comment by JatinBisht2308** [comment]
@narayan954 sir I will work on the UI of the new post. Please assign it to me and also add gssoc23 label.

**Comment by narayan954** [comment]
> @narayan954 sir I will work on the UI of the new post. Please assign it to me and also add gssoc23 label.

Sure, create an issue mentioning the same and your approach :)

**Comment by JatinBisht2308** [comment]
@narayan954 sir I will mention it all tommorow morning is it fine sir.

**Comment by narayan954** [comment]
> @narayan954 sir I will mention it all tommorow morning is it fine sir.

It's fine take your time, no need to rush :)

**Comment by mohitparmar1** [comment]
hey @narayan954, I would like to work on this issue.
I have some ideas related to UI.

**Comment by narayan954** [comment]
> hey @narayan954, I would like to work on this issue. I have some ideas related to UI.

great, create the issue regarding the part you want to improve
